### PR TITLE
[QA-1939] feat: add cypress login test

### PR DESCRIPTION
**Pull request by andradepradorafael**
### What is the new behavior introduced by this PR?
The new behavior introduced by this PR is the addition of an end-to-end (e2e) automated login test using Cypress within the Azion Console Kit. This test verifies the login functionality efficiently and reliably by following the Arrange, Act, Assert (AAA) notation.

### Does this PR introduce breaking changes?
- [X] No
- [ ] Yes

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -32,7 +32,7 @@ coverage
 .env
 .env.*
 !.env.sample
-cypress.env.*
+cypress.env.json
```
Comment (matheus-sunderhus) [inline .gitignore:35]: cypress.env.*
!cypress.env.exmaple.json

Comment (matheus-sunderhus) [reply .gitignore:35]: deixar o padrão de exclusão mais genérico e o de inclusão mais específico 👍🏻

**Review by matheus-sunderhus — COMMENTED** [review]
Ajuste de git ignore

File: .gitignore (hunk 1 above)
Comment (andradepradorafael) [reply .gitignore:35]: Done!

File: .gitignore [hunk 2]
```diff
@@ -33,6 +33,7 @@ coverage
 .env.*
 !.env.sample
 cypress.env.*
+!cypress.env.exmaple.json
```
Comment (matheus-sunderhus) [inline .gitignore:36]: typo, faltou o A no exAmple

Comment (andradepradorafael) [reply .gitignore:36]: Done!

**Review by matheus-sunderhus — APPROVED** [review]
LGTM

**Review by william-tome — APPROVED** [review]

**Review by alisson-marcelo-azion — CHANGES_REQUESTED** [review]
E2E tests are failing

**Review by alisson-marcelo-azion — APPROVED** [review]

**Review by higorjasonduarte — APPROVED** [review]

**Pull request merged**